docs/octant-plugin-installation.md

```diff
@@ -114,15 +114,17 @@ based on your environment and move the binary to OCTANT_PLUGIN_PATH.
 Now, you are supposed to see Octant is running together with antrea-octant-plugin via URL http://(IP or $HOSTNAME):80.
 
 Note:
-1. In Antrea v0.8.1, the Traceflow UI is a separate Octant plugin called antrea-traceflow-plugin.
-Starting with v0.9.0, the Traceflow UI will be merged into antrea-octant-plugin. When deploying Octant as a Pod using
-image antrea/octant-antrea-ubuntu:v0.8.1, you already have access to the alpha version of the Traceflow UI.
+1. In Antrea v0.8.2, the Traceflow UI is a separate Octant plugin called antrea-traceflow-plugin, but it has been
+merged into antrea-octant-plugin on master branch. To get the latest version of Traceflow UI, please build image
```

Comment:
s/on master branch/on the master branch since then

Reply:
Done. I also moved this part to the Note2 of the first section for deploying as a Pod.